Automattic/sensei
# Add helper functions for HPPS settings

Resolves #7322

Added a class with a few constants and static methods to make it easier to check current settings.
Unfortunately, in current state it is impossible to use some methods inside the Sensei class or in Settings class (that will be result in infinite recursion).
Depending on our needs, we can reconsider that and partially resolve the issue.

## Proposed Changes
* Add helper functions for HPPS settings.

## Testing Instructions

1. Enable experimental UI.
2. Go to Sensei LMS > Settings > Experimental Features.
3. No errors/warnings expected.


## Pre-Merge Checklist
